src/Wallabag/ApiBundle/Controller/AnnotationRestController.php

```diff
@@ -39,7 +39,7 @@ public function getAnnotationsAction(Entry $entry)
      * @ApiDoc(
      *      requirements={
      *          {"name"="ranges", "dataType"="array", "requirement"="\w+", "description"="The range array for the annotation"},
-     *          {"name"="quote", "dataType"="string", "required"=false, "description"="Optional, quote for the annotation"},
+     *          {"name"="quote", "dataType"="string", "required"=true, "description"="Quote for the annotation"},
```

Comment:
Changing this to required will force all client to update their code.
Maybe we should remind to add it in the changelog as a _small_ BC.

What do you think @wallabag/core?

Reply:
I don't know of any client that supports annotations, apart from https://github.com/di72nn/wallabag-api-wrapper from @di72nn 

Fine by me to add it in the changelog of course.

Reply:
Anyway I'm not even sure what's the purpose of this field.

Reply:
So as far as anyone here knows this field has no use (in current implementation), everything works ok if it's `null` (it did work ok previously, it does now if it's nullable in DB) - why not make it nullable?
Otherwise you make effectively useless field mandatory.

Also it is already nullable with wallabag.it (postgres?).

Reply:
> Also it is already nullable with wallabag.it (postgres?).

Not yet postgres

Reply:
The frontend does not seem to use the `quote` text, so the quote is now optionnal. If the quote is empty, an empty string is saved